util/chplenv/printchplenv.py

```diff
@@ -91,7 +91,8 @@
     ChapelEnv('CHPL_HOST_MEM', INTERNAL, 'hostmem'),
     ChapelEnv('CHPL_TARGET_MEM', INTERNAL, 'tgtmem'),
     ChapelEnv('CHPL_MEM', RUNTIME | LAUNCHER | DEFAULT, 'mem'),
```

Comment:
Should this be split into `CHPL_HOST_MEM` and `CHPL_TARGET_MEM`?

Reply:
Yeah that was the direction that got the most traction so that eg an unsupported combo for the compiler wouldn't prevent you from using `jemalloc` in the runtime.